docs/docs/app/visual.mdx

```diff
@@ -14,6 +14,12 @@ The Visual Editor enables non-technical users to design A/B tests on their site
 
 In March, 2023 we released version 2.0 of our Visual Editor with many improvements.
 
+:::note
+
+Visual editor features may not work optimally on client-side rendered apps (e.g., React.js, Vue.js). Consider using our [Feature Flags](/app/features) feature for smoother integration. Contact support for further assistance.
```

Comment:
I think our overuse of the word `feature` is making this a little confusing.  Also, I'd probably put the full support email address as well.

```suggestion
The Visual Editor may not work optimally on client-side rendered apps (e.g. React.js, Vue.js). Consider using [Feature Flags](/app/features) instead for smoother integration. Contact support@growthbook.io if you have any questions.
```